pom.xml

```diff
@@ -49,19 +49,19 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>ch.qos.logback</groupId>
-                <artifactId>logback-access</artifactId>
-                <version>1.4.14</version>
+                <groupId>ch.qos.logback.access</groupId>
+                <artifactId>common</artifactId>
+                <version>2.0.3</version>
```

Comment:
Move this down below the other `logback-core`, as the ordering is alphabetical by groupId.